spoti-vote/src/components/Login.jsx

```diff
@@ -17,7 +17,16 @@ let pictures = {
 	}
 };
 
+
+
+
 class Login extends Component {
+	componentDidMount() {
+		if (window.location.search !== '') {
+			window.location.search = '';
```

Comment:
isnt this used with the query-string npm plugin?

Reply:
Whenever someone wants to access a nonexisting room he will be linked back to the login page, hower if he has a token or a name in the url it will stay there, with this it will be removed.

Still searching for a better way to do this